cashctrl_ledger/ledger.py

```diff
@@ -53,17 +53,110 @@ def _single_account_balance():
         """
         raise NotImplementedError
 
-    def account_chart():
+    def account_chart(self) -> pd.DataFrame:
+        """Retrieves the account chart from a remote CashCtrl instance, formatted to the pyledger schema."""
```

Comment:
For coherence, I would write
```
"""
Retrieves the account chart from a remote CashCtrl instance, formatted to the pyledger schema.
"""
```
Instead of:
```
"""Retrieves the account chart from a remote CashCtrl instance, formatted to the pyledger schema."""
```